frontend/src/app/spot/styles/sass/components/modal.sass

```diff
@@ -0,0 +1,80 @@
+@import './modal-close-button'
+@import './modal-overlay'
+@import './modal-overrides'
+
+.spot-modal
+  position: relative
+  display: flex
+  flex-direction: column
+  align-items: stretch
+  background: white
+  pointer-events: all
+
+  border-radius: $spot-spacing-0-25
+  width: 40rem
+
+  max-width: calc(100vw - 2rem)
+  max-height: calc(100vh - 2rem)
+
+  overflow-y: auto
+
+  @media (max-width: 680px)
+    width: calc(100vw - 2rem)
+    max-height: calc(100vh - 5rem)
+
+    @media (max-height: 480px)
+      max-height: calc(100vh - 2rem)
+      max-width: calc(100vw - 6rem)
+
+  &_wide
+    width: 70rem
+    min-height: 40vh
+
+  &_fullscreen
+    width: 100vw
+    height: 100vh
```

Comment:
I don't think this is in use at all, it isn't part of the specification either way I think